From patchwork Sun Nov 21 17:03:45 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Peter Robinson <pbrobinson@gmail.com>
X-Patchwork-Id: 1557782
X-Patchwork-Delegate: matthias.bgg@gmail.com
Return-Path: <u-boot-bounces@lists.denx.de>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: bilbo.ozlabs.org;
	dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20210112 header.b=Nfi74Zyb;
	dkim-atps=neutral
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.denx.de
 (client-ip=85.214.62.61; helo=phobos.denx.de;
 envelope-from=u-boot-bounces@lists.denx.de; receiver=<UNKNOWN>)
Received: from phobos.denx.de (phobos.denx.de [85.214.62.61])
	(using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
	 key-exchange X25519 server-signature RSA-PSS (4096 bits) server-digest
 SHA256)
	(No client certificate requested)
	by bilbo.ozlabs.org (Postfix) with ESMTPS id 4HxxZZ0gF9z9t5G
	for <incoming@patchwork.ozlabs.org>; Mon, 22 Nov 2021 04:04:04 +1100 (AEDT)
Received: from h2850616.stratoserver.net (localhost [IPv6:::1])
	by phobos.denx.de (Postfix) with ESMTP id 51CEA830F6;
	Sun, 21 Nov 2021 18:03:55 +0100 (CET)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=u-boot-bounces@lists.denx.de
Authentication-Results: phobos.denx.de;
	dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.b="Nfi74Zyb";
	dkim-atps=neutral
Received: by phobos.denx.de (Postfix, from userid 109)
 id 4CFC3832FC; Sun, 21 Nov 2021 18:03:53 +0100 (CET)
X-Spam-Checker-Version: SpamAssassin 3.4.2 (2018-09-13) on phobos.denx.de
X-Spam-Level: 
X-Spam-Status: No, score=-2.1 required=5.0 tests=BAYES_00,DKIM_SIGNED,
 DKIM_VALID,DKIM_VALID_AU,DKIM_VALID_EF,FREEMAIL_FROM,SPF_HELO_NONE,
 SPF_PASS autolearn=ham autolearn_force=no version=3.4.2
Received: from mail-wm1-x336.google.com (mail-wm1-x336.google.com
 [IPv6:2a00:1450:4864:20::336])
 (using TLSv1.3 with cipher TLS_AES_128_GCM_SHA256 (128/128 bits))
 (No client certificate requested)
 by phobos.denx.de (Postfix) with ESMTPS id 0E61D810ED
 for <u-boot@lists.denx.de>; Sun, 21 Nov 2021 18:03:50 +0100 (CET)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=pbrobinson@gmail.com
Received: by mail-wm1-x336.google.com with SMTP id
 d72-20020a1c1d4b000000b00331140f3dc8so11665334wmd.1
 for <u-boot@lists.denx.de>; Sun, 21 Nov 2021 09:03:50 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20210112;
 h=from:to:cc:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=r7Q3ah4vxfXTUmICZhdGzoXO9pob06QAXnQE48bMDRE=;
 b=Nfi74Zyb9X7zyuCsJpujekTt8Od4uuoqGtn48tTNmTmJsWW9iV/+4QhAKCHNxrrmaT
 qRkFAM+/62a6yXqY5LvE8SuaGgMa3/pQDJJy9AivhrVL+FS3PO2Zt47GRZFgEaHSkZmb
 ANmw+JgHZVqPtSPdwFOGqwzmp9b7+gI/Ig17fh+kjCfL2H/0Y0n1iuRwznM7CCKK3Ana
 YWbtxS0vkT6uMxJFdKqoNWL0F2USa9tDvpxFevZflxb5+oaGxGRkUXS1jtop0/ojNeUk
 9uDoL0Vv8Z4u4Kbb1CIcX9J+h3NEwCyzMFXy3HEBT0gWC+bxjODH8qOB1aWiKb65/IWp
 kbXw==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20210112;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=r7Q3ah4vxfXTUmICZhdGzoXO9pob06QAXnQE48bMDRE=;
 b=ukPl/MLkqFWd/QqNPHLJonHgsnIL70sLP/i+GhTF7yj2iR+LbIEuDH6u4G469m7hNO
 jHdCXg5ikz5TvHkgfq0osShSLpsWhxztOogswKaOl5WvFzIX0pgK3At0CAQyo5Iw421d
 qY4yFLmVmppvgjq5JMm67/jjh+cPu/r2pV55X7m38RvNS8OZPRjGug5MQsgAIjfdqjFp
 OQ6WpsKeV6iYR58vH+c94MTRW8ZDypn8oduxLM0gr6kWYcRLn1AeL6GQoIJpkzpd3y8g
 RwQS+VQ8/1moonYbIGIjtSFiCin8Bq8vlgqvUvd3jKs1LXVjNLd0DJi963ZafZgIHijd
 +V3A==
X-Gm-Message-State: AOAM531MhbLJwpw+nj/VT4x1S0PdvKkhoFm0krkEa5+LPlowK4rmjuVE
 VuMrwhZy2YEjPSIksph/tS4=
X-Google-Smtp-Source: 
 ABdhPJxbdLZkafdyasxKGEHOw9bvqmq7Fyx2p2hAqhE9dadsNaGtKyOREv/9mJ6iIifjb4OrEVryOQ==
X-Received: by 2002:a05:600c:378b:: with SMTP id
 o11mr21715495wmr.157.1637514229247;
 Sun, 21 Nov 2021 09:03:49 -0800 (PST)
Received: from morpheus.home.roving-it.com
 (a.7.4.5.7.2.7.b.4.1.9.2.a.3.0.0.1.8.6.2.1.1.b.f.0.b.8.0.1.0.0.2.ip6.arpa.
 [2001:8b0:fb11:2681:3a:2914:b727:547a])
 by smtp.googlemail.com with ESMTPSA id o4sm21813664wmq.31.2021.11.21.09.03.48
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Sun, 21 Nov 2021 09:03:48 -0800 (PST)
From: Peter Robinson <pbrobinson@gmail.com>
To: Matthias Brugger <mbrugger@suse.com>,
	u-boot@lists.denx.de
Cc: Peter Robinson <pbrobinson@gmail.com>
Subject: [PATCH 1/2] rpi: Update the Raspberry Pi doucmentation URL
Date: Sun, 21 Nov 2021 17:03:45 +0000
Message-Id: <20211121170346.347343-1-pbrobinson@gmail.com>
X-Mailer: git-send-email 2.33.1
MIME-Version: 1.0
X-BeenThere: u-boot@lists.denx.de
X-Mailman-Version: 2.1.37
Precedence: list
List-Id: U-Boot discussion <u-boot.lists.denx.de>
List-Unsubscribe: <https://lists.denx.de/options/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=unsubscribe>
List-Archive: <https://lists.denx.de/pipermail/u-boot/>
List-Post: <mailto:u-boot@lists.denx.de>
List-Help: <mailto:u-boot-request@lists.denx.de?subject=help>
List-Subscribe: <https://lists.denx.de/listinfo/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=subscribe>
Errors-To: u-boot-bounces@lists.denx.de
Sender: "U-Boot" <u-boot-bounces@lists.denx.de>
X-Virus-Scanned: clamav-milter 0.103.2 at phobos.denx.de
X-Virus-Status: Clean

The Raspberry Pi Foundation has updated their documentation so update
the URL to the latest place to find the HW device revision codes.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 board/raspberrypi/rpi/rpi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/raspberrypi/rpi/rpi.c b/board/raspberrypi/rpi/rpi.c
index 55afaa54d9..e7bcee3746 100644
--- a/board/raspberrypi/rpi/rpi.c
+++ b/board/raspberrypi/rpi/rpi.c
@@ -72,7 +72,7 @@ struct msg_get_clock_rate {
 #endif
 
 /*
- * https://www.raspberrypi.org/documentation/hardware/raspberrypi/revision-codes/README.md
+ * https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#raspberry-pi-revision-codes
  */
 struct rpi_model {
 	const char *name;

From patchwork Sun Nov 21 17:03:46 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Peter Robinson <pbrobinson@gmail.com>
X-Patchwork-Id: 1557783
X-Patchwork-Delegate: matthias.bgg@gmail.com
Return-Path: <u-boot-bounces@lists.denx.de>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: bilbo.ozlabs.org;
	dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20210112 header.b=mopvZ/a0;
	dkim-atps=neutral
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.denx.de
 (client-ip=85.214.62.61; helo=phobos.denx.de;
 envelope-from=u-boot-bounces@lists.denx.de; receiver=<UNKNOWN>)
Received: from phobos.denx.de (phobos.denx.de [85.214.62.61])
	(using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
	 key-exchange X25519 server-signature RSA-PSS (4096 bits))
	(No client certificate requested)
	by bilbo.ozlabs.org (Postfix) with ESMTPS id 4HxxZh03Sqz9t5G
	for <incoming@patchwork.ozlabs.org>; Mon, 22 Nov 2021 04:04:11 +1100 (AEDT)
Received: from h2850616.stratoserver.net (localhost [IPv6:::1])
	by phobos.denx.de (Postfix) with ESMTP id 5A7068331E;
	Sun, 21 Nov 2021 18:03:58 +0100 (CET)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=u-boot-bounces@lists.denx.de
Authentication-Results: phobos.denx.de;
	dkim=pass (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.b="mopvZ/a0";
	dkim-atps=neutral
Received: by phobos.denx.de (Postfix, from userid 109)
 id ACABC810ED; Sun, 21 Nov 2021 18:03:53 +0100 (CET)
X-Spam-Checker-Version: SpamAssassin 3.4.2 (2018-09-13) on phobos.denx.de
X-Spam-Level: 
X-Spam-Status: No, score=-2.1 required=5.0 tests=BAYES_00,DKIM_SIGNED,
 DKIM_VALID,DKIM_VALID_AU,DKIM_VALID_EF,FREEMAIL_FROM,SPF_HELO_NONE,
 SPF_PASS autolearn=ham autolearn_force=no version=3.4.2
Received: from mail-wr1-x434.google.com (mail-wr1-x434.google.com
 [IPv6:2a00:1450:4864:20::434])
 (using TLSv1.3 with cipher TLS_AES_128_GCM_SHA256 (128/128 bits))
 (No client certificate requested)
 by phobos.denx.de (Postfix) with ESMTPS id 34173830F6
 for <u-boot@lists.denx.de>; Sun, 21 Nov 2021 18:03:50 +0100 (CET)
Authentication-Results: phobos.denx.de;
 dmarc=pass (p=none dis=none) header.from=gmail.com
Authentication-Results: phobos.denx.de;
 spf=pass smtp.mailfrom=pbrobinson@gmail.com
Received: by mail-wr1-x434.google.com with SMTP id b12so28142146wrh.4
 for <u-boot@lists.denx.de>; Sun, 21 Nov 2021 09:03:50 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20210112;
 h=from:to:cc:subject:date:message-id:in-reply-to:references
 :mime-version:content-transfer-encoding;
 bh=Ck6Y933+5vinfoQY1AQPdGxK5p8+add34hJkwYpA2t0=;
 b=mopvZ/a0Rnxwn0AKiQm1L9bJpP6xUSaMemeNXf0YqpftTSGYKmPgs7ReD4hoKZiHEP
 yAYOskB6wAkavpQCJXrUNZ8fXLqUuVvX/6MxbzCZ5FJnlNC6KqxE+pS+0N05wi9+Z5HA
 9CF1CWxgw8XzkUPiTvoy7U3/q2xhKl4zqHnmINNz08c8fyjRHcPdW6Nqt+9rtnPqizWB
 06nY7FkMYoXYAT1jFpdpGwyxB21q9kitOHABWKIil5soin6rOMOIDgv/k1h85Oz56FR+
 CHTZXyJTqhrL2eczw0CSipg9tSWVshmmtXToXKgRX5NGpt7DQX5Xvfeq6IVlzEAKeFnW
 80IA==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20210112;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:in-reply-to
 :references:mime-version:content-transfer-encoding;
 bh=Ck6Y933+5vinfoQY1AQPdGxK5p8+add34hJkwYpA2t0=;
 b=NqWpyRhPdJ9op+t4C4jwtWbTznUZkUowj2YZZooiD+f0JulW8yLTueM3UkoGf3yNWL
 9Dl1FF7sC4oHLUS2CLclujNe+YgImdiA93/Dx0teK3DgFo3A3kTl6QLiS6eCJsMkNx8a
 Hbu5uR1Gfi1fKyKuXnzBVuNJYQuThl27Fkl4rQ25n0JNb3G4cCMatx7q/Q76uXvKS+mV
 nQN2xYwTbhFUA2tDxb9ePEs1oK+S6K86r3mqXQmNsL/GPv5SZyixIZ9vLihIRZkIc3Ah
 QuJ/qPN9CSoHwUJ/UjtO8T/EHgfUI3oKfwIGrYwKGKHdqgosmq3tLDwCpktrkRMPM8yL
 093w==
X-Gm-Message-State: AOAM530YGkYNwawwIsYCPIrHYl3ghsV+s0LTU5ydQ6skhuf7J7KLGgj5
 ftCYoJkrIvlyYAA7D2tZ9P0=
X-Google-Smtp-Source: 
 ABdhPJwKb79/iu7T7anhnMMB69LV9rpqaYh7so4zBenphcofi9TjlvElQJ+oNL/uS1ycyj2z7iqixg==
X-Received: by 2002:a5d:588b:: with SMTP id
 n11mr30307204wrf.344.1637514229828;
 Sun, 21 Nov 2021 09:03:49 -0800 (PST)
Received: from morpheus.home.roving-it.com
 (a.7.4.5.7.2.7.b.4.1.9.2.a.3.0.0.1.8.6.2.1.1.b.f.0.b.8.0.1.0.0.2.ip6.arpa.
 [2001:8b0:fb11:2681:3a:2914:b727:547a])
 by smtp.googlemail.com with ESMTPSA id o4sm21813664wmq.31.2021.11.21.09.03.49
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Sun, 21 Nov 2021 09:03:49 -0800 (PST)
From: Peter Robinson <pbrobinson@gmail.com>
To: Matthias Brugger <mbrugger@suse.com>,
	u-boot@lists.denx.de
Cc: Peter Robinson <pbrobinson@gmail.com>
Subject: [PATCH 2/2] rpi: Add identifier for the new RPi Zero 2 W
Date: Sun, 21 Nov 2021 17:03:46 +0000
Message-Id: <20211121170346.347343-2-pbrobinson@gmail.com>
X-Mailer: git-send-email 2.33.1
In-Reply-To: <20211121170346.347343-1-pbrobinson@gmail.com>
References: <20211121170346.347343-1-pbrobinson@gmail.com>
MIME-Version: 1.0
X-BeenThere: u-boot@lists.denx.de
X-Mailman-Version: 2.1.37
Precedence: list
List-Id: U-Boot discussion <u-boot.lists.denx.de>
List-Unsubscribe: <https://lists.denx.de/options/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=unsubscribe>
List-Archive: <https://lists.denx.de/pipermail/u-boot/>
List-Post: <mailto:u-boot@lists.denx.de>
List-Help: <mailto:u-boot-request@lists.denx.de?subject=help>
List-Subscribe: <https://lists.denx.de/listinfo/u-boot>,
 <mailto:u-boot-request@lists.denx.de?subject=subscribe>
Errors-To: u-boot-bounces@lists.denx.de
Sender: "U-Boot" <u-boot-bounces@lists.denx.de>
X-Virus-Scanned: clamav-milter 0.103.2 at phobos.denx.de
X-Virus-Status: Clean

The Raspberry Pi Foundation released the new Zero 2 W which we
want to detect, so we can detect the correct device tree file name.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 board/raspberrypi/rpi/rpi.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/board/raspberrypi/rpi/rpi.c b/board/raspberrypi/rpi/rpi.c
index e7bcee3746..ff69607d3c 100644
--- a/board/raspberrypi/rpi/rpi.c
+++ b/board/raspberrypi/rpi/rpi.c
@@ -157,6 +157,11 @@ static const struct rpi_model rpi_models_new_scheme[] = {
 		DTB_DIR "bcm2711-rpi-4-b.dtb",
 		true,
 	},
+	[0x12] = {
+		"Zero 2 W",
+		DTB_DIR "bcm2837-rpi-zero-2.dtb",
+		false,
+	},
 	[0x13] = {
 		"400",
 		DTB_DIR "bcm2711-rpi-400.dtb",
